src/index.html

```diff
@@ -1,13 +1,478 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>The MET</title>
+    <link
+    rel="shortcut icon"
+    href="./images/icons/the_met_favicon.jpg"
+    type="image/x-icon">
```

Comment:
Need to fix and check it everywhere
```suggestion
    <link
      rel="shortcut icon"
      href="./images/icons/the_met_favicon.jpg"
      type="image/x-icon"
    >
```